test/res.sendFile.js

```diff
@@ -615,7 +615,7 @@ describe('res', function(){
         .get('/')
         .end(function(err, res){
           res.text.should.equal('<p>{{user.name}}</p>');
-          res.headers.should.have.property('content-type', 'text/html; charset=UTF-8');
+          should(res.headers).have.property('content-type', 'text/html; charset=UTF-8');
```

Comment:
I don't mind either way, we're just trying to get fixes in place earlier than later. If you know of a better, equivalent fix then I say go for it :+1: